HISTORY.md

```diff
@@ -1,6 +1,6 @@
 # History
 
-## Unreleased
+## 2.1.0 (2020-12-05)
```

Comment:
Okay, let's bump to 3.0.0, might save some surprises for people.

```suggestion
## 3.0.0 (2020-12-05)
```

Would you like me to do the release or would you like to?

Reply:
🚀